.ci/run_tests.sh

```diff
@@ -19,4 +19,6 @@ run_oldtests
 
 install_nvim
 
+run_single_includes_test
```

Comment:
why does this come after `install_nvim` ? can it go before `build_nvim` (line 10) to fail the build faster?